src/types/DiaryCardProps.ts

```diff
@@ -0,0 +1,5 @@
+export type DiaryCardProps = {
```

Comment:
types that are needed for a page or component you can define them inside the component